testing/regress/ecl/sets3.ecl

```diff
@@ -17,7 +17,7 @@
 
 gavLib := service
     set of integer4 getPrimes() : eclrtl,pure,library='eclrtl',entrypoint='rtlTestGetPrimes',oldSetFormat;
-    set of integer4 getFibList(const set of integer4 inlist) : eclrtl,pure,library='eclrtl',entrypoint='rtlTestFibList',newset;
+    set of integer4 getFibList(const set of integer4 inlist) : eclrtl,pure,library='eclrtl',entrypoint='rtlTestFibList',newset,time;
```

Comment:
This syntax seems to imply that the decision about whether a function call is worth timing is made when by the publisher of the function rather than the user of the function - which seems a little counter-intuitive to me.